src/WebJobs.Script/Description/Workers/WorkerFunctionInvoker.cs

```diff
@@ -91,11 +91,23 @@ protected override async Task<object> InvokeCore(object[] parameters, FunctionIn
             };
 
             string invocationId = context.ExecutionContext.InvocationId.ToString();
-            _logger.LogTrace($"Sending invocation id:{invocationId}");
+            ScriptInvocationResult result = new();
+
+            _logger.LogTrace("Sending invocation id: '{id}", invocationId);
             await _functionDispatcher.InvokeAsync(invocationContext);
-            var result = await invocationContext.ResultSource.Task;
+
+            try
+            {
+                result = await invocationContext.ResultSource.Task;
```

Comment:
Is accessing the ResultSource.Task enough? Is it possible for line 97 to throw a TaskCancelledException/OperationCancelled exception ?

Reply:
I don't think so, the invocation task is the `ResultSource.Task` and that's what we're trying to handle, I can go back to including line 97 if we need to